guides/rest_doc.md

```diff
@@ -741,6 +745,150 @@ kintoneUtility.rest.getFormLayout({
+| param.app | Number | Yes | The app ID.
+| param.isPreview | Boolean | Optional |  **Default:** `false`. Set `true` if you want to get pre-live setting.
+| param.isGuest | Boolean | Optional | **Default:** `false`. Set `true` if the app belongs to a guest space.
+
+#### Response
+```
+Promise Object
+```
+#### Sample
+```js
+kintoneUtility.rest.getCustomize({
+  app: 542,
+}).then(function(response) {
+  console.log(response);
+}).catch(function(error) {
+  console.log(error);
+});
+```
+
+## <a name="putCustomize"> kintoneUtility.rest.putCustomize(param)
+* Update the information of JavaScrit/CSS files used to customize an app.
+
+#### Parameter
+
+| Name | Data type | Required | Description
+|:-----------|:------------:|:------------:|:------------
+| param | Object | Yes |  |
+| param.app | Number | Yes | The app ID.
+| param.scope | String | Optional | The target to apply this setting: `ALL`, `ADMIN`, or `NONE`.
+| param.desktop | Object | Optional | Customize files for the desktop app.
+| param.desktop.js | Object[] | Optional | JavaScript files information.
+| param.desktop.js[].type | String | Yes | The type of the file: `URL` or `FILE`.
+| param.desktop.js[].url | String | Optional | The URL of the file. Required when `URL` is specified as the type.
+| param.desktop.js[].file | Object | Optional | The information of the uploaded file.  Required when `FILE` is specified as the type.
+| param.desktop.js[].file.fileKey | String | Yes | The file key of the uploaded file.
+| param.desktop.css | Object[] | Optional | CSS files information.
+| param.desktop.css[].type | String | Yes | The type of the file: `URL` or `FILE`.
+| param.desktop.css[].url | String | Optional | The URL of the file. Required when `URL` is specified as the type.
+| param.desktop.css[].file | Object | Optional | The information of the uploaded file.  Required when `FILE` is specified as the type.
+| param.desktop.css[].file.fileKey | String | Yes | The file key of the uploaded file.
+| param.mobile | Object | Optional | Customize files for the mobile app.
+| param.mobile.js | Object[] | Optional |
+| param.mobile.js[].type | String | Yes | The type of the file: `URL` or `FILE`.
+| param.mobile.js[].url | String | Optional | The URL of the file. Required when `URL` is specified as the type.
+| param.mobile.js[].file | Object | Optional | The information of the uploaded file.  Required when `FILE` is specified as the type.
+| param.mobile.js[].file.fileKey | String | Yes | The file key of the uploaded file.
+| param.revision | Number | Optional | The expected revision number. If the value does not match, an error will occur and the record will not be updated. If the value is not specified or is -1, the revision number will not be checked.
+| param.isGuest | Boolean | Optional | **Default:** `false`. Set `true` if the app belongs to a guest space.
+ 
+#### Response
+```
+Promise Object
+```
+#### Sample
+```js
+kintoneUtility.rest.putCustomize({
```

Comment:
Because customize is a verb, so the function name should be changed
putCustomize => updateCustomization